config/dev/config.edn

```diff
@@ -5,11 +5,11 @@
                                    :user     "postgres"
                                    :password "postgres"}
  :framework.app/web-server        {:port  3000
-                                    :join? false}
+                                   :join? false}
  :framework.db.storage/migration  {:store                :database
-                                   :migration-dir        "resources/migrations"
+                                   :migration-dir        "resources/migrations/dev"
```

Comment:
@g-krisztian what are the advantages of having different folders for `dev` and `test` migrations? To be able to seed some testing data for tests? I don't think it's worth the troubles, better to do it in the test itself.

Reply:
Maybe you need different migrations for dev and test, but I'm sure you need different for prod

Reply:
Could you pls give an example why it might be necessary? The ability to change migration directory for different environments might be useful, but a lot of projects will be totally fine having just one migrations directory, and the framework shouldn't force that on the project. So I don't think it should be here.

Reply:
I don't think it's forced here. It's forced in the template :wink: but can be removed there. If I remember well there was a discussion on Frankie to replace migratus just because it's not supporting profiles, so it's a solution. But still you can use the same directory for all your profiles.

Reply:
I would suggest using the same `:migration-dir` and `:migration-table-name` for all environments. If we need to differentiate database states between environments we should use a different database.

Reply:
I've changed it

Reply:
sorry, wrong branch :D

Reply:
Fixed

Reply:
Thank you!